frontend-react/src/hooks/UseLookupTable.ts

```
@@ -18,43 +17,34 @@ export interface TableAttributes {
 
 export async function getLatestVersion(
     tableName: LookupTables
-): Promise<TableAttributes | null> {
```

changes here revolve around throwing on errors rather than returning null